pkg/ebpf/perf_batching.go

```diff
@@ -0,0 +1,139 @@
+// +build linux_bpf
+
+package ebpf
+
+import (
+	"fmt"
+	"time"
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/network"
+	bpflib "github.com/iovisor/gobpf/elf"
+)
+
+const (
+	TCPCloseBatchSize = 5
+	maxNumberBatches  = 1024
```

Comment:
Does this limit come from BPF ?

Reply:
Initially I thought of getting the number of CPU cores (as seen by the kernel) via the proc filesystem somehow, but [it was mentioned that we tried this before with no success](https://github.com/DataDog/datadog-agent/pull/5472/files/771d1faea8ceca70746e60a0da6b14fa4cffef2a#r423336826). So basically we overshoot here the number of CPU cores. 1024 was suggested as being the maximum number of entries in a map (though that doesn't seem the case looking at the [documentation](https://man7.org/linux/man-pages/man2/bpf.2.html)), but I think it's a good enough number to use as the default.